core/storage/collection/gae_models.py

```diff
@@ -328,7 +328,7 @@ def compute_models_to_commit(
         )
 
         # Here we use cast because the additional_models is list of BaseModels
-        # and we want to hint the mypy that this is CollectionRightsModel.
+        # and we want to hint the MyPy that this is CollectionRightsModel.
```

Comment:
You missed "the" --> "to" in my previous comment.

Please pay attention to detail and make sure you've addressed everything before marking something as "Done". Otherwise we have unnecessary review cycles.

Reply:
change `"the" --> "to"` is added